Tweaks to cicero-scm to make Halons work, and have Heat Uptake be in W/m^2 (#52)
openscm/openscm-runner: CHANGELOG.rst

    - Fixed: any bug fixes
    - Security: in case of vulnerabilities.

2021-09-17
-------------------

Fixed
~~~~~~~

> - (`#52 <https://github.com/openscm/openscm-runner/pull/52>` _) Fixing CICERO-SCM bugs with converting halon units and handling of very long scenario names

Reviewer: ```suggestion
- (`#52 <https://github.com/openscm/openscm-runner/pull/52>`_) Fixed CICERO-SCM bugs related to converting halon units and handling of very long scenario names
```

Perfect except for one missing space (feel free to keep or ignore my other rephrasing as you see fit)
